limited.go

```diff
@@ -60,6 +60,14 @@ func (lvs *LimitedValueStore) GetValue(ctx context.Context, key string, opts ...
 	return lvs.ValueStore.GetValue(ctx, key, opts...)
 }
 
+// GetValue returns ErrNotSupported
+func (lvs *LimitedValueStore) SearchValue(ctx context.Context, key string, opts ...ropts.Option) (<-chan []byte, error) {
+	if !lvs.KeySupported(key) {
+		return nil, routing.ErrNotFound
```

Comment:
Again, it may be more accurate here to just return no error and a closed channel. That's what we'd return if we, e.g., search the DHT and fail to find anything.